### PR TITLE
[18.0][MIG] purchase_order_type: Migration to 18.0

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Review by CasVissers-360ERP — COMMENTED** [review]
Functional review

File: purchase_order_type/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Order Type",
+    "version": "18.0.1.0.2",
```
Comment (DRING02) [inline purchase_order_type/__manifest__.py]:
```suggestion
    "version": "18.0.1.0.0",
```

**Review by DRING02 — APPROVED** [review]

File: purchase_order_type/__manifest__.py (hunk 1 above)
Comment (StefanRijnhart) [reply purchase_order_type/__manifest__.py]: Dang, thank you! Fixed now.

**Review by DRING02 — APPROVED** [review]

**Review by celm1990 — APPROVED** [review]
LGTM

**Pull request merged**